app/models/ssh_execution.rb

```diff
@@ -1,4 +1,7 @@
 class SshExecution
+  class_attribute :executioner
```

Comment:
I think you mean `executor` now you're talking about someone who kills for living :D